src/Core/Components/List/FluentAutocomplete.razor.cs

```diff
@@ -171,6 +174,26 @@ public override bool Multiple
     [Parameter]
     public bool ShowOverlayOnEmptyResults { get; set; } = true;
 
+    /// <summary>
+    /// If true, the options list will be rendered with virtualization. This is normally used in conjunction with
+    /// scrolling and causes the option list to fetch and render only the data around the current scroll viewport.
+    /// This can greatly improve the performance when scrolling through large data sets.
+    ///
+    /// If you use <see cref="Virtualize"/>, you should supply a value for <see cref="ItemSize"/> and must
+    /// ensure that every row renders with the same constant height.
+    ///
+    /// Generally it's preferable not to use <see cref="Virtualize"/> if the amount of data being rendered
+    /// is small or if you are using pagination.
```

Comment:
We don't offer support for pagination in the search result list so please change this line.

People might think it is supported otherwise